amazon_eks/README.md

```diff
@@ -16,7 +18,17 @@ Monitoring EKS requires that you set up the Datadog integrations for:
 
 along with integrations for any other AWS services you're running with EKS (e.g., [ELB](https://docs.datadoghq.com/integrations/amazon_elb/))
 
+### Log Collection
+
+**Available for Agent >6.0**
+
+The setup is exactly the same as for Kubernetes. To start collecting logs from all your containers, only two environment variables needs to be set on your Datadog Agent.
```

Comment:
@NBParis which env variables should be used? is there pre-defined values that we recommend for those two variables?

Reply:
Issue is, if we start to give more here, we will write the full Kubernetes documentation here. Which is why I did as for metrics and linked to the relevant Kubernetes section of the documentation.

Let me know if you think we should rewrite everything there.